### PR TITLE
Add coding convention with Kotlin for android

### PR DESCRIPTION
**Pull request by longth-1474**

### CODE REVIEW

**Comment by tuyenbq-0446** [comment]
OK

File: vn/android/coding_convention_android_kotlin.md [hunk 1]
```diff
@@ -0,0 +1,1312 @@
+# Coding Style Guide
```
Comment (ngocnd0242) [inline vn/android/coding_convention_android_kotlin.md:1]: @longth-1474
https://github.com/framgia/coding-standards/blob/2e7e176d0fa5111fe1d0e1b67db25a92afc7dbbc/vn/android/coding_convention_android_kotlin.md
có 12 ký tự lạ � em nhé!

**Review by ngocnd0242 — CHANGES_REQUESTED** [review]
When viewing md file,  appeared 12 special characters �, please help me check it.

**Comment by longth-1474** [comment]
@ngocnd0242 e đã fix các kỹ thứ lạ rồi ạ. A check lại giúp e với ạ

**Review by ngocnd0242 — APPROVED** [review]

**Pull request merged**